recipe/meta.yaml

```diff
@@ -11,11 +11,11 @@ source:
     - define_OLD_BUGGY_REMQUO.patch
 
 build:
-  number: 1
+  number: 2
   run_exports:
-    # so name changes in bugfix revisions.  Pin to bugfix revision.
+    # ABI has been stable across minor versions since 8.0.0
     #    https://abi-laboratory.pro/tracker/timeline/proj/
-    - {{ pin_subpackage('proj', max_pin='x.x.x') }}
+    - {{ pin_subpackage('proj', max_pin='x.x') }}
```

Comment:
@speleo3, please check that this is what you were hoping for in #144.

Reply:
Looks good, thank you!